graph/src/util/lfu_cache.rs

```diff
@@ -94,9 +100,11 @@ impl<K: Clone + Ord + Eq + Hash + Debug, V: CacheWeight + Default> LfuCache<K, V
                 );
             }
             Some(entry) => {
+                let old_weight = entry.weight;
```

Comment:
A bit simpler (removes temporary):

```
Some(entry) => {
   self.total_weight -= entry.weight;
   self.total_weight += weight;
   entry.weight = weight;
   entry.value = value;
}
```

Reply:
I wrote it that way since I find it a little easier to follow what is happening there - I bet the compiler generates pretty much the same code for the two variants.